l10n_ro_balance_confirmation/models/res_partner.py

```diff
@@ -0,0 +1,44 @@
+# ©  2008-2022 Terrabit
+# See README.rst file on addons root folder for license details
+
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _name = "res.partner"
+    _inherit = "res.partner"
+
+    def get_debit_credit(self):
```

Comment:
valoarea nu este calculata si standard ?

Reply:
Am incercat cu campul "debit" din res.partner initial, dar in unele cazuri imi aduce 0, chiar daca are sold.

Reply:
In ce cazuri  functia standard nu functioneaza corect ?